pinc/CharacterSelector.inc

```diff
@@ -1,241 +1,137 @@
 <?php
-include_once($relPath.'site_vars.php'); // $wikihiero_dir $utf8_site
-include_once($relPath.'misc.inc'); // attr_safe(), html_safe()
+
+include_once($relPath.'prefs_options.inc'); // get_user_proofreading_font()
 
 class CharacterSelector
 {
-    public function __construct()
+    function __construct ()
     {
+        $this->picker_set =
```

Comment:
This does not include the set of UTF8 characters from the prior version, we should retain that functionality.

Reply:
I started on this but it seems like it could be better to change the way the data is stored back to unicode code points rather than utf8 strings. Some characters are difficult to distinguish in a text editor for example for example U+00D0 capital eth, U+0110 D with stroke, and U+0189 African D. For the same reason it would be better to have a tool-tip text (possibly empty) for every character stored with it rather than in a separate array.
These considerations would apply if the data is hard coded or if we were to store configurable data in the database.

Reply:
<nod> I think keeping them as unicode code points make sense.

I wonder if for the tool tip we want to use the Unicode character name. They aren't exactly sexy but they are specific and unambiguous -- [IntlChar::charName](https://www.php.net/manual/en/intlchar.charname.php), for instance. That would add a new dependency (`php-intl`) but I'm OK with that.

Reply:
Great idea, that will save a lot of work.